examples/websocket-relay/relay-app/src/components/node/Edge.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  import { Edge } from "svelvet";
+
+  export let arrowDirection: string = 'left'
+</script>
+
+<Edge let:path labelColor="white">
+	<defs>
+		<marker
+			id="arrow"
+			viewBox="0 0 10 10"
+			refX="11"
+			refY="5"
+			markerWidth="6"
+			markerHeight="6"
+			orient="auto-start-reverse"
+      fill="#000000"
+		>
+			<path d="M 0 0 L 10 5 L 0 10 z" />
+		</marker>
+	</defs>
+	<path
+		d={path}
+    {...(arrowDirection === 'right' ? { 'marker-end': "url(#arrow)" } : { 'marker-start': "url(#arrow)" })}
```

Comment:
Nice! Glad adding the arrow worked out. 🙂

Reply:
Yeah! I think I had them working for a while before I realized, but the Anchor positioning was hiding them from me 😅 just had to do some light styling to get them looking similar to before 👍🏼